manifests/namespaced-install/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+- ../base
+
+resources:
+- namespace.yaml
+
+namespace: kubeflow
+
+patchesStrategicMerge:
+- workflow-controller-configmap.yaml
+- ml-pipeline-persistenceagent-deployment-patch.yaml
+- ml-pipeline-scheduledworkflow-deployment-patch.yaml
+
+vars:
+  - name: NAMESPACE
```

Comment:
This is a smart way of keeping the namespace the same as the namespace directive in kustomization.yaml. Good job.